### DISCUSSION TITLE
"Forcing" dark mode?

### DISCUSSION THREAD

**Discussion by royabradley** [alshedivat/al-folio#563] [Q&A] [answered]
Is there a simple way to "force" dark mode to be the default, even on systems running in light mode?

**Comment by SEllersgaard** [comment]
I would really like an answer to this as well. 
How do you set the default mode to dark?

**Comment by adityajain93** [comment]
Same! I don't know how to do this but super interested

**Reply by adityajain93** [reply, marked as answer]
if (userPref && (userPref('(prefers-color-scheme: dark)').matches || userPref('(max-width: 768px)').matches || true)) {

**Comment by adityajain93** [comment]
if (userPref && (userPref('(prefers-color-scheme: dark)').matches || userPref('(max-width: 768px)').matches || true)) {

**Comment by george-gca** [comment]
See #670
